Makefile

```diff
@@ -47,7 +47,10 @@ install-dev:
 
 jlab-watch:
 	($(CONDA_ACTIVATE) ${ENV_NAME}; \
-		jupyter lab --watch --port 8234)
+		jupyter lab \
+			--watch \
+			--port 8234 \
+			--ServerApp.token=)
```

Comment:
+1, should we remove that once tokens are supported ?

Reply:
Authentication will be enabled on my next commit.